app/controllers/comments_controller.rb

```diff
@@ -1,7 +1,7 @@
 class CommentsController < ApplicationController
+  before_filter :authenticate_user!
   def create
-    # ajout if logged in apres
-   Comment.create!(title:params[:name],content:params[:description],product_id:params[:product_id],user_id:current_user.id)
-   redirect_to product_path(params[:product_id])
+    Comment.create!(title:params[:name],content:params[:description],product_id:params[:product_id],user_id:current_user.id)
```

Comment:
Line is too long. [124/80]<br>Space missing after colon.<br>Space missing after comma.